packages/esm-outpatient-app/src/patient-search/patient-scheduled-visits.component.tsx

```diff
@@ -16,46 +25,84 @@ enum priority {
   EMERGENCY = 'Emergency',
 }
 
-const PatientScheduledVisits: React.FC<PatientSearchProps> = ({ toggleSearchType }) => {
+enum visitType {
+  RECENT = 'Recent',
+  FUTURE = 'Future',
+}
+
+const ScheduledVisits: React.FC<{ visits; isLoading; visitType }> = ({ visits, isLoading, visitType }) => {
   const { t } = useTranslation();
-  const isTablet = useLayoutType() === 'tablet';
-  const [showRecentPriority, setShowRecentPriority] = useState(false);
-  const [futurePrioritySwitcherValue, setFuturePrioritySwitcherValue] = useState(0);
-  const [futureVisitsIndex, setFutureVisitsIndex] = useState(0);
+  const [prioritySwitcherValue, setSwitcherValue] = useState(0);
+  const [visitsIndex, setVisitsIndex] = useState(0);
+  const [show_priority, setShowPriority] = useState(false);
```

Comment:
```suggestion
  const [hasPriority, setHasPriority] = useState(false);
```
The project uses `camelCase` for consistency purposes we should stick to it